docs/source/tutorials/e2e_flow.rst

```diff
@@ -25,222 +25,239 @@ from the community seemlessly with torchtune.
-
-.. code-block:: bash
-
-    tune run lora_finetune_single_device \
-    --config llama2/7B_lora_single_device \
-    checkpointer.checkpoint_dir=<checkpoint_dir> \
-    tokenizer.path=<checkpoint_dir>/tokenizer.model \
-    checkpointer.output_dir=<checkpoint_dir>
-
-
-Once training is complete, you'll see the following in the logs.
-
-.. code-block:: bash
-
-    [_checkpointer.py:473] Model checkpoint of size 9.98 GB saved to <checkpoint_dir>/hf_model_0001_0.pt
+We'll fine-tune using our
+`single device LoRA recipe <https://github.com/pytorch/torchtune/blob/main/recipes/lora_finetune_single_device.py>`_
+and use the standard settings from the
+`default config <https://github.com/pytorch/torchtune/blob/main/recipes/configs/llama3_2/3B_lora_single_device.yaml>`_.
 
-    [_checkpointer.py:473] Model checkpoint of size 3.50 GB saved to <checkpoint_dir>/hf_model_0002_0.pt
+This will fine-tune our model using a ``batch_size=2`` and ``dtype=bfloat16``. With these settings the model
+should have a peak memory usage of ~16GB and total training time of around 2-3 hours for each epoch.
 
-    [_checkpointer.py:484] Adapter checkpoint of size 0.01 GB saved to <checkpoint_dir>/adapter_0.pt
+.. code-block:: text
 
+    $ tune run lora_finetune_single_device --config llama3_2/3B_lora_single_device
+    Setting manual seed to local seed 3977464327. Local seed is seed + rank = 3977464327 + 0
+    Hint: enable_activation_checkpointing is True, but enable_activation_offloading isn't. Enabling activation offloading should reduce memory further.
+    Writing logs to /tmp/torchtune/llama3_2_3B/lora_single_device/logs/log_1734708879.txt
+    Model is initialized with precision torch.bfloat16.
+    Memory stats after model init:
+            GPU peak memory allocation: 6.21 GiB
+            GPU peak memory reserved: 6.27 GiB
+            GPU peak memory active: 6.21 GiB
+    Tokenizer is initialized from file.
+    Optimizer and loss are initialized.
+    Loss is initialized.
+    Dataset and Sampler are initialized.
+    Learning rate scheduler is initialized.
+    Profiling disabled.
+    Profiler config after instantiation: {'enabled': False}
+    1|3|Loss: 1.943998098373413:   0%|                    | 3/1617 [00:21<3:04:47,  6.87s/it]
+
+Congrats on training your model! Let's take a look at the artifacts produced by torchtune. A simple way of doing this is by running `tree -a path/to/outputdir`, which should show something like the tree below.
+There are 4 types of folders:
+
+1) **recipe_state**: Holds recipe_state.pt with the information necessary to restart the last intermediate epoch. For more information, please check our deep-dive :ref:`Checkpointing in torchtune <understand_checkpointer>`.;
+2) **logs**: Defined in your config in metric_logger;
+3) **epoch_{}**: Contains your new trained model weights plus all original files of the model, except the checkpoints, making it easy for you to choose an specific epoch to run inference on or push to a model hub;
```

Comment:
i made some changes to it in checkpointer, based on evans comments. Probably work copying/pasting it here.

Reply:
skip one extra line after 3).
![image](https://github.com/user-attachments/assets/7ca7295f-c59f-4f1e-bc80-769f854c606b)